library/client_events.js

```diff
@@ -104,7 +104,7 @@ module.exports = client => {
         let prChan = client.guilds.get(newMember.guild.id).channels.find(chan => chan.name === 'practice-room-chat')
         if (prChan == null) {
           client.log('Cannot find #practice-room-chat!')
-        } else if (restwo.permitted_channels.includes(newMember.voiceChannelID)) {
+        } else if (restwo.permitted_channels.includes(newMember.voiceChannelID) && !(newMember.selfMute && newMember.selfDeafened)) {
```

Comment:
`mute`, not `selfMute`.

If they're server muted and self-deafened we still want this to happen, I think.